<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductSubPage</class>
 <widget class="QWidget" name="ProductSubPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(243, 188, 46);</string>
  </property>
  <widget class="QLabel" name="label_category_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>100</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Consolas&quot;;
color: rgb(50, 50, 50);</string>
   </property>
   <property name="text">
    <string>CategoryName</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_category">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Consolas&quot;;
color: rgb(50, 50, 50);</string>
   </property>
   <property name="text">
    <string>Category:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>250</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Consolas&quot;;
color: rgb(50, 50, 50);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>531</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(50, 50, 50);
font: 12pt &quot;Consolas&quot;;</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_desc">
    <attribute name="title">
     <string>Description</string>
    </attribute>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>501</width>
       <height>231</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid white;
font: 12pt &quot;Consolas&quot;;
color: rgb(50, 50, 50);</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_rate">
    <attribute name="title">
     <string>Rate</string>
    </attribute>
    <widget class="QTableWidget" name="table">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>531</width>
       <height>171</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Consolas&quot;;
color: rgb(0, 0, 0);</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>171</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_stock">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Consolas&quot;;
color: rgb(50, 50, 50);</string>
   </property>
   <property name="text">
    <string>Stock: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_seller_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>130</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Consolas&quot;;
color: rgb(50, 50, 50);</string>
   </property>
   <property name="text">
    <string>SellerName</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_stock_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Consolas&quot;;
color: rgb(50, 50, 50);</string>
   </property>
   <property name="text">
    <string>stockNum</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_seller">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Consolas&quot;;
color: rgb(50, 50, 50);</string>
   </property>
   <property name="text">
    <string>Seller:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_pdName">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Consolas&quot;;
color: rgb(50, 50, 50);</string>
   </property>
   <property name="text">
    <string>ProductName</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_addToCart">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>250</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/addtocart.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_SF">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>160</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Consolas&quot;;
color: rgb(50, 50, 50);</string>
   </property>
   <property name="text">
    <string>Shipping Fee: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_SF_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>160</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Consolas&quot;;
color: rgb(50, 50, 50);</string>
   </property>
   <property name="text">
    <string>Shipping Fee</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_rating_1">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Consolas&quot;;
color: rgb(50, 50, 50);</string>
   </property>
   <property name="text">
    <string>Rating:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_rating_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>190</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Consolas&quot;;
color: rgb(50, 50, 50);</string>
   </property>
   <property name="text">
    <string>Rating</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_price">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>250</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Consolas&quot;;
color: rgb(50, 50, 50);</string>
   </property>
   <property name="text">
    <string>Price</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
